.github/workflows/buildAndTest.yml

```diff
@@ -0,0 +1,71 @@
+# Copyright 2022 The StableHLO Authors.
+
+name: Build and Test
+
+on:
+  pull_request:
+    branches: [ main ]
+  push:
+    branches: [ main ]
+  schedule:
+    # Always regenerate once every day
+    - cron:  '* 12 * * *'
+  workflow_dispatch:
+
+# Ensure that only a single job or workflow using the same
+# concurrency group will run at a time. This would cancel
+# any in-progress jobs in the same github workflow and github
+# ref (e.g. refs/heads/main or refs/pull/<pr_number>/merge).
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+# Build using CMake and run tests.
+# Use Cached LLVM to improve build times.
+# Current setup only builds on linux.
```

Comment:
Is this comment still up-to-date given that we say `os-arch: maci-x86_64` below?

Reply:
deleted.